cli/replicaset/roles.go

```diff
@@ -0,0 +1,43 @@
+package replicaset
+
+import "fmt"
+
+type Role string
```

Comment:
It's still not clear why we need this, maybe we could just to work with `string`? To be clear, we don't expect this type to ever change.

Reply:
Deleted custom role.